src/model-viewer-base.ts

```diff
@@ -320,7 +320,6 @@ export default class ModelViewerElementBase extends UpdatingElement {
 
   [$onResize](e: {width: number, height: number}) {
     this[$scene].setSize(e.width, e.height);
-    this[$needsRender]();
```

Comment:
Was this taken out because some other part of the base implementation invokes the re-render?

Reply:
I was following the prior PR's method here, but you're right, there's a better way.